augraphy/augmentations/lowinkline.py

```diff
@@ -32,26 +32,6 @@ def __init__(
         self.use_consistent_lines = use_consistent_lines
         self.noise_probability = noise_probability
 
-        inconsistent_transparency_line = lambda x: random.randint(0, 255)
```

Comment:
In this commit, i removed all `np.vectorize` an replace it by np array broadcasting method. Using an image with size of 1200x1200, the speed improved from 1.6 to 0.2 seconds in periodiclines and 4 to 0.2 seconds in randomlines.

Reply:
Wow.  Nice update!